src/desktop/request.rs

```diff
@@ -31,6 +31,31 @@ where
     Err(ResponseError),
 }
 
+#[cfg(feature = "backend")]
+impl<T> Response<T>
+where
+    T: for<'de> Deserialize<'de> + Type,
+{
+    pub fn ok(inner: T) -> Self {
+        Self::Ok(inner)
+    }
+
+    pub fn cancelled() -> Self {
+        Self::Err(ResponseError::Cancelled)
+    }
+
+    pub fn other() -> Self {
+        Self::Err(ResponseError::Other)
+    }
+}
+
+#[cfg(feature = "backend")]
+impl Response<BasicResponse> {
+    pub fn empty() -> Self {
```

Comment:
I wonder if we  can use Default here instead?

Reply:
I dont thin the semantics fit